docs/integrations/databases/neon.mdx

```diff
@@ -119,72 +104,100 @@ export const db = drizzle(sql, {
 ```
 </CodeBlockTabs>
 
-### Generate and run database migrations
+### Push the schema to the database
 
-Use the `drizzle-kit` package to generate and run database migrations.
+To load the schema into the database, create a `drizzle.config.ts` file at the root of your project and add the following configuration:
 
-1. At the root of your project, create a `drizzle.config.ts` and add the following configuration:
-    ```typescript {{ filename: 'drizzle.config.ts' }}
-    import { defineConfig } from "drizzle-kit";
-    import { loadEnvConfig } from "@next/env";
+```typescript {{ filename: 'drizzle.config.ts' }}
+import { defineConfig } from "drizzle-kit";
+import { loadEnvConfig } from "@next/env";
 
-    loadEnvConfig(process.cwd());
+loadEnvConfig(process.cwd());
 
-    if (!process.env.DATABASE_URL) {
-      throw new Error("DATABASE_URL must be a Neon postgres connection string");
-    }
+if (!process.env.DATABASE_URL) {
+  throw new Error("DATABASE_URL must be a Neon postgres connection string");
+}
 
-    export default defineConfig({
-      dialect: "postgresql",
-      dbCredentials: {
-        url: process.env.DATABASE_URL,
-      },
-      schema: "./app/db/schema.ts",
-      out: "./migrations",
-    });
-    ```
-1. Generate and run the database migrations by running the following commands:
-    ```sh {{ filename: 'terminal' }}
-    npx drizzle-kit generate
-    npx drizzle-kit migrate
-    ```
+export default defineConfig({
+  dialect: "postgresql",
+  dbCredentials: {
+    url: process.env.DATABASE_URL,
+  },
+  schema: "./app/db/schema.ts",
+});
+```
+
+Run the following command to push the schema to the database:
```

Comment:
Let's have this use a numbered list to keep it consistent across our docs - if a `<Step>` has more than one instruction in it, we use a numbered list.

Let's also rearrange this sentence to follow the same structure as the previous one: Why, then how (take action). Why = "To push the schema to the database", How = "Run the following command"

```suggestion
1. To push the schema to the database, run the following command:
```